src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Tasks/ManagedResourceParserTests.cs

```diff
@@ -320,6 +320,27 @@ void BuildLibraryWithResources (string path)
 			}
 		}
 
+		void CompareFilesIgnoreRuntimeInfoString (string file1, string file2)
+		{
+			FileAssert.Exists (file1);
+			FileAssert.Exists (file2);
+			var content1 = File.ReadAllText (file1);
+			var content2 = File.ReadAllText (file2);
+			// This string is only generated when running on mono, replace with a new line that will be stripped when comparing.
+			string versionInfo = "//     Runtime Version:4.0.30319.42000";
```

Comment:
Should we make this a `Regex` just in case the version number changes? If this just removed an entire line that matches: `^\/\/\s*Runtime Version:`

Reply:
Not sure this requires a *regex*.  Removing any line starting with `//     Runtime Version:` would likely be sufficient.

Reply:
Updated to use a regex, the regex replace behavior seems like it'll be easier to read than the alternative.